### ISSUE TITLE
kendo.editor.linkOpenInNewWindow -> No translation found

### ISSUE THREAD

**Issue by meriturva** [telerik/kendo-angular#2916]
There is something strange with latest version of the editor ("@progress/kendo-angular-editor": "1.1.0")
No way to get translation string on xlf file (extration just dosen't add new string):

`No translation found for "6304412528521452300" ["7ddbe5ff681ec56e0a1d47f55baf03aae43143f0"] ("Open link in new window" - "kendo.editor.linkOpenInNewWindow").`

As workaround just add it manually:

```
<trans-unit id="7ddbe5ff681ec56e0a1d47f55baf03aae43143f0" datatype="html">
        <source>Open link in new window</source>
        <target/>
        <context-group purpose="location">
          <context context-type="sourcefile">../node_modules/@progress/kendo-angular-editor/dist/es2015/editor.component.d.ts</context>
          <context context-type="linenumber">66</context>
        </context-group>
        <note priority="1" from="description">The caption for the checkbox for opening the link in a new window in the createLink
                dialog.
            </note>
        <note priority="1" from="meaning">
                 kendo.editor.linkOpenInNewWindow</note>
      </trans-unit>
```

**Comment by PetarMetodiev** [comment]
Related to https://github.com/telerik/kendo-angular-messages/issues/110

**Comment by siyka-marinova** [comment]
Fixed in `kendo-angular-editor v1.2.0`
